habs_portal/static/json/stations.json

```diff
@@ -0,0 +1,11 @@
+{
+  "stations" : [
+    {
+      "id":"tollsps",
+      "name": "Toledo Low Service Station",
+      "cateogories" : ["temperature", "nutrients"],
+      "lat": 40,
+      "lon": -80
+    }
```

Comment:
this will come from the services ultimately, but for now I can fill it out with like one or two for testing.